libc/utils/hdrgen/main.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+#
+# ===- Generate headers for libc functions  ------------------*- python -*--==#
```

Comment:
I *think* a recent policy change llvm-wide is to skip the emacs style language includes...

https://discourse.llvm.org/t/is-c-in-header-files-still-relevant/83124/15

Reply:
That discussion and change seems only to relate to conventions for C++ source files.
That document doesn't say anything at all about "file headers" for Python source files.

Regardless, I simply copied the style of the adjacent files.  If there is some convention, either across LLVM or specific to libc, that source files should follow, then please change the existing files en masse to meet that convention.  Contributors should be able to take existing files as a model for new files.